DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System;
+
+    public static class DisplayStringHelper
+    {
+        private const string Divider = " / ";
+        private static readonly string[] Units = { "B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB" };
+
+        public static string GenerateNumberWithLimitDisplayString(int number, int limit)
+        {
+            return limit == -1 ? number.ToString() : number + Divider + limit;
+        }
+
+        public static string GenerateBytesLimitDisplayString(long number, long limit)
+        {
+            return GenerateBytesDisplayString(number) + Divider + GenerateBytesDisplayString(limit);
+        }
+
+        private static string GenerateBytesDisplayString(long byteCount)
+        {
+            if (byteCount == 0)
+            {
+                return 0 + Units[0];
+            }
+
+            var bytes = Math.Abs(byteCount);

```

Comment:
Hmm.. Do we need the absolute value?

Reply:
This is needed to calculate the value without throwing errors. Log(negative number) is an imaginary number, which makes no sense in this context. The method should ideally never be called with a negative number as it doesn't really make sense in terms of storage space.

Reply:
It doesn't seem right to take an absolute value in case the value is negative - we should either throw or return "0B" (rather than 1B for -1)

Can we return "0B" for now as for some reason negative values are handled in the old system - I'll run this past Kevin tomorrow morning. 🙏